docs/bigquery/snippets.py

```diff
@@ -320,6 +319,24 @@ def test_create_table_without_schema(client, to_delete):
     # [END bigquery_create_table_without_schema]
 
     to_delete.insert(0, table)
+    table_id = 'my_table'
```

Comment:
ISTM it would be better to move this assignment to the top of the method and then use it for initializing / comparing the table ref. rather than repeating the literal `'my_table'`.

Reply:
Yeah, let's be consistent within this method at least.

I'd also be okay with repeating the literal `'my_table'` everywhere, since that makes it 100% clear in the sample that we are talking about string IDs.